.buildkite/build.rayci.yml

```diff
@@ -0,0 +1,39 @@
+group: build
+steps:
+  - label: ":tapioca: build: wheel {{matrix}}"
+    instance_type: medium
+    commands:
+      - ./ci/build/build-manylinux-ray.sh
+      - ./ci/build/build-manylinux-wheel.sh {{matrix}}
+      - if [ "$BUILDKITE_PULL_REQUEST" != "false" ]; then exit 0; fi
+      - export PATH=/opt/python/cp38-cp38/bin:$$PATH
+      - pip install -q aws_requests_auth boto3
+      - ./ci/env/env_info.sh
+      # Upload to branch directory.
+      - python .buildkite/copy_files.py --destination branch_wheels --path ./.whl
+      # Upload to latest directory.
+      - if [ "$BUILDKITE_BRANCH" == "master" ]; then python .buildkite/copy_files.py --destination wheels --path ./.whl; fi
```

Comment:
you would need to remove it counterpart from civ1. either this copy step, or the entire step that builds the wheel.

otherwise there would be a write-write race condition, and potentially create failures or inconsistent results.

Reply:
good catch